### DISCUSSION TITLE
FEAT: Add dark/light

### DISCUSSION THREAD

**Discussion by yacdanyCR** [ArslanYM/StarterHive#206] [Q&A] [answered]
According to the requirements requested for this feature

1. Add toggle between light mode and dark mode.
2. We should be able to switch between each using a button in the navbar.
- 
Everything is done , this disccussion is to show the work and receive and feedback if I have to update something or if a specific color is going to be provided and if not so the task is done.
![Screenshot (64)](https://github.com/ArslanYM/StarterHive/assets/77762960/538c89fc-26fd-4cd6-b7b6-6843699190f4)
![Screenshot (66)](https://github.com/ArslanYM/StarterHive/assets/77762960/90267b52-af89-438d-a547-e6b84c3d7925)

**Comment by ArslanYM** [comment, marked as answer]
@yacdanyCR How about we keep the current theme as dark and just work on the light mode? What do you think about that?

**Reply by ArslanYM** [reply]
Not that I dont like the black version that youve created. Im asking for your opinion.

**Reply by yacdanyCR** [reply]
@ArslanYM  Thats sound good , the default theme is already dark so you're right, I'm going to work in light mode.

**Reply by ArslanYM** [reply]
Great! Thankyou for your contributions. If you have any other questions feel free to ask me , otherwise you can mark the above comment as an answer for  this discussion..

**Reply by ArslanYM** [reply]
@yacdanyCR do you have any other doubts? If not you can mark the above comment as the answer. Thankyou.

**Reply by ArslanYM** [reply]
@yacdanyCR Can you mark the above marked answer as this discussion's answer. Would be a huge help.
